sup3r/pipeline/forward_pass_cli.py

```diff
@@ -68,21 +68,30 @@ def from_config(ctx, config_file, verbose):
               verbose=verbose)
 
     exec_kwargs = config.get('execution_control', {})
-    log_prefix = config.get('log_file_prefix', None)
-    hardware_option = exec_kwargs.get('option', 'local')
+    log_pattern = config.get('log_pattern', None)
+    if log_pattern is not None:
+        if '.log' not in log_pattern:
+            log_pattern += '.log'
+        if '{node_index}' not in log_pattern:
+            log_pattern = log_pattern.replace('.log', '_{node_index}.log')
+
     logger.debug('Found execution kwargs: {}'.format(exec_kwargs))
+    hardware_option = exec_kwargs.pop('option', 'local')
     logger.debug('Hardware run option: "{}"'.format(hardware_option))
 
     sig = signature(ForwardPassStrategy)
     strategy_kwargs = {k: v for k, v in config.items()
                        if k in sig.parameters.keys()}
     strategy = ForwardPassStrategy(**strategy_kwargs)
 
-    for i in range(strategy.nodes):
+    node_index = config.get('node_index', None)
+    nodes = [node_index] if node_index is not None else range(strategy.nodes)
```

Comment:
I'm not sure if this is the right place but can we str(node_index).zfill(5) somewhere?